dnf/yum/rpmtrans.py

```diff
@@ -409,6 +414,12 @@ def _script_start(self, key):
         complete = self.complete_actions if self.total_actions != 0 and self.complete_actions != 0 \
             else 1
         total = self.total_actions if self.total_actions != 0 and self.complete_actions != 0 else 1
+
+        # In case rpm trigger runs scriptlet of some other package that is not in the transaction
+        if isinstance(key, str) and key != pkg.name:
+            # TODO(amatej): This is bad I think, but we have only the name :(
+            pkg = self.base._sack.query().installed().filterm(name=key)[0]
```

Comment:
_sack() returns list of installed packages that were installed before the transactions
it's not changing during the transaction

consider just printing the package name without any lookups